pkg/util/ecs/common.go

```diff
@@ -36,8 +36,17 @@ func ListContainersInCurrentTask() ([]*containers.Container, error) {
 		log.Error("Unable to get the container list from ecs")
 		return cList, err
 	}
+
+	filter, err := containers.GetSharedMetricFilter()
+	if err != nil {
+		log.Warnf("Unable to get container filter. All containers in ECS Task will be processed, err: %v", err)
```

Comment:
`ListContainersInCurrentTask` is called continuously so this warning might become spammy - let's switch to debug wdyt?

Reply:
I believe it should be spammy, customers would be paying for containers they don't want.